Rakefile

```diff
@@ -2,7 +2,7 @@ require "bundler/gem_tasks"
 require "rake/testtask"
 require "rubocop/rake_task"
 
-Rake::TestTask.new(:test) do |t|
+Rake::TestTask.new(:rubocop_md_tests) do |t|
```

Comment:
Why prefixing with `rubocop_md`? This is the Rakefile for the project, so theme is implied.